delfin/drivers/hitachi/vsp/rest_handler.py

```diff
@@ -0,0 +1,208 @@
+# Copyright 2020 The SODA Authors.
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+import threading
+
+import requests
+import six
+from oslo_log import log as logging
+
+from delfin import cryptor
+from delfin import exception
+from delfin.drivers.hitachi.vsp import consts
+
+LOG = logging.getLogger(__name__)
+STATUS_200 = 200
+STATUS_201 = 201
+STATUS_202 = 202
+STATUS_204 = 204
+STATUS_401 = 401
+
+
+class RestHandler(object):
```

Comment:
`RestHandler` is a rest client which can implement some specific functions for vsp, so `RestHandler` should inherit `RestClient`.
And then all the `self.rest_client.xxx` could be `self.xxx`

Reply:
has change RestHandler inherit RestClient